src/api/kube.ts

```diff
@@ -1096,6 +1105,23 @@ export class KubeHelper {
     }
   }
 
+  async checkCRDUpdate(name: string, yamlFilePath: string) {
+    const { spec: crdFileSpec } = this.safeLoadFromYamlFile(yamlFilePath) as V1beta1CustomResourceDefinition
+    const { spec: crdClusterSpec } = await this.getCrd(name)
+    const { validation: { openAPIV3Schema : { properties: crdFileProps = '' }= {} } = {} } = crdFileSpec
+    const { validation: { openAPIV3Schema : { properties: crdClusterProps = '' }= {} } = {} } = crdClusterSpec
+
+    if (crdFileSpec.version !== crdClusterSpec.version) {
```

Comment:
As far as I remember `version` field is deprecated. 
To get the correct version is needed to get version where `storage: true` in `versions` array,
https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/